### DISCUSSION TITLE
updating docs_theme to newer bootstrap

### DISCUSSION THREAD

**Discussion by mostafaei2002** [encode/django-rest-framework#9402] [Potential Issue]
Hey there. The current docs_theme uses bootstrap 2.1 and is kind of annoying to work with especially for mobile devices. The navbar is kinda broken on mobile devices you have to hold nav items to open drop downs which is not ideal.
It's not necessary to upgrade to a new bootstrap version. A simpler solution would be to slightly tweak the bootstrap JS files or add some additional scripts to the web page. 
Completely upgrading to bootstrap v5 also should be easy to implement since the HTML in total is around 200 lines.

**Comment by cclauss** [comment]
Duplicate of:
* #7327
* #7411 
* #9028

**Reply by mostafaei2002** [reply]
The first 2 were mostly focused on bootstrap inside DRF itself and they didn't really mention the docs page, at the end, it was stalled.
I did not come across the 3rd one in my search though unfortunately. I was planning to create an issue then a PR to work on the docs page issue so I created this discussion beforehand.
Should I close this as duplicate and continue on the 3rd one? Or just continue from here?

**Comment by cclauss** [comment]
I think starting a pull request is vital.  Is there a way to create smaller PRs that are easier to review and merge or must this be done in large PRs that touch numerous files like the previous attempts?

**Reply by mostafaei2002** [reply]
No I do it in small steps it's totally fine. I'll try my best to create more commits. This is not like the previous version too though since the whole html code is not more than 250 lines.
Is customizing the Bootstrap an option too? If any customization is needed i can setup sass and we can customize some things from there.
I will fix the html structure first though then we can discuss that part.

**Comment by mostafaei2002** [comment]
I made a PR but there is no one reviewing it. There is a problem in the CI/CD pipeline causing my PR to be rejected. Is that possibly the reason why no one is reviewing the PR? Thanks in advance
